packages/gatsby-cli/src/init-starter.js

```diff
@@ -147,18 +147,22 @@ module.exports = async (starter: string, options: InitOptions = {}) => {
 
   const urlObject = url.parse(rootPath)
   if (urlObject.protocol && urlObject.host) {
+    trackError(`NEW_PROJECT_NAME_MISSING`)
```

Comment:
i'm wondering if we should move into an enum or leverage typescript types for these strings. Should we also namespace these like `STARTER_NEW_PROJECT_NAME_MISSING` or would this be overkill as we won't have that many events.

Reply:
These are also mostly freeform events